README.md

```diff
@@ -46,119 +53,121 @@ Department name:
+    * 11 - Waitress
 
+* Arline
+    * 1 - Air Crew
+    * 2 - Airline Captain
+    * 3 - Airline Pilot
+    * 4 - Airport Manager
+    * 5 - Analyst
+    * 6 - Chief Pilot
+    * 7 - Traffic Manager
 
 # Setup
+
 * Required:
-  * Docker
-  * Postgres SQL
-  * Java 11 or higher
+    * Docker
+    * Postgres SQL
+    * Java 11 or higher
 * Optional
-  * PgAdmin4
+    * PgAdmin4
 
-The application uses Postgres SQL Server, which runs in a Docker container. Please execute the following command to create the containers for Postgres SQL and PgAdmin4
+The application uses Postgres SQL Server, which runs in a Docker container. Please execute the
+following command to create the containers for Postgres SQL and PgAdmin4
 
 * Docker container postgress command line:
-  * docker pull postgres
-  * docker docker run -p 5432:5432 -d \                                              
-    -e POSTGRES_PASSWORD=password \
-    -e POSTGRES_USER=postgres \
-    -e POSTGRES_DB=stripe-example \
-    -v pgdata:/var/lib/postgresql/data \
-    postgres
-  
-
-* Docker container PgAdmin4: 
-  * docker pull dpage/pgadmin4
-  * docker run --name my-own-postgres -e POSTGRES_PASSWORD=postgresmaster -p 5432:5432 -d postgres
-  
+    * docker pull postgres
+    * docker docker run -p 5432:5432 -d \                                              
+      -e POSTGRES_PASSWORD=password \
+      -e POSTGRES_USER=postgres \
+      -e POSTGRES_DB=stripe-example \
+      -v pgdata:/var/lib/postgresql/data \
+      postgres
+
+
+* Docker container PgAdmin4:
+    * docker pull dpage/pgadmin4
+    * docker run --name my-own-postgres -e POSTGRES_PASSWORD=postgresmaster -p 5432:5432 -d postgres
+
 * Docker compose file: docker-compose-salary-calculator.yml
-  * Go to the folder: PolandSalaryCalculator
-  * execute: docker-compose -f docker-compose-salary-application.yml up
+    * Go to the folder: PolandSalaryCalculator
+    * execute: docker-compose -f docker-compose-salary-application.yml up
 
 * Create a table in PostgreSQL by executing the create table query.
 
-  * CREATE TABLE IF NOT EXISTS public.data_salary_calculator
-(
-    id integer NOT NULL DEFAULT nextval('salary_calculator_spring_id_seq'::regclass),
-    pension_zus numeric,
-    disability_zus numeric,
-    sickness_zus numeric,
-    total_zus numeric,
-    health numeric,
-    annual_gross numeric,
-    tax numeric,
-    net_monthly numeric,
-    annual_net numeric,
-    gross_monthly numeric,
-    job_title character varying COLLATE pg_catalog."default",
-    CONSTRAINT salary_calculator_spring_pkey PRIMARY KEY (id)
-)
+    * CREATE TABLE IF NOT EXISTS public.data_salary_calculator
+      (
+      id integer NOT NULL DEFAULT nextval('salary_calculator_spring_id_seq'::regclass),
+      pension_zus numeric,
+      disability_zus numeric,
+      sickness_zus numeric,
+      total_zus numeric,
+      health numeric,
+      annual_gross numeric,
+      tax numeric,
+      net_monthly numeric,
+      annual_net numeric,
```

Comment:
What do you think about keeping consistency? Previously you had `annual_gross` and next you have `annual_net` so consistent it would be `monthly_net`.